packages/@vue/cli-service/lib/config/base.js

```diff
@@ -37,6 +37,10 @@ module.exports = (api, options) => {
         .publicPath(options.publicPath)
 
     webpackConfig.resolve
+      // This plugin can be removed once we switch to Webpack 6
+      .plugin('pnp')
+        .use({ ...require('pnp-webpack-plugin') })
```

Comment:
The spread is a small trick required for `webpack-chain` (it mutates the plugin instance, but since the PnP plugin is supposed to be the same instance for every config it crashes when webpack-chain tries to override its previous registration).